### ISSUE TITLE
`envyable install` fails on vanilla install

### ISSUE THREAD

**Issue by omnilord** [philnash/envyable#9]
I've been trying out Envyable for a couple of hours and have been running into a number of problems.  I am still trying to document a larger problem, but I found this one while working on that.

Fairly straight forward, running `envyable install`  (with or without Rails) results in this error message.  Tried locally on a Mac, and again on a fresh install of ruby and rails on an Ubuntu server, both with the same results.

```
[02:56 ubuntu@server:~/projects/env_test]$ envyable install
 /home/ubuntu/.rbenv/versions/2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- ./lib/envyable/cli (LoadError)
    from /home/ubuntu/.rbenv/versions/2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
    from /home/ubuntu/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/envyable-1.1.0/bin/envyable:4:in `<top (required)>'
    from /home/ubuntu/.rbenv/versions/2.3.1/bin/envyable:23:in `load'
    from /home/ubuntu/.rbenv/versions/2.3.1/bin/envyable:23:in `\<main\>'
```

Envyable version 1.1.0

Mac has Ruby 2.3.0, Rails 4.2.6
Ubuntu has Ruby 2.3.1, Rails 5.0.0.1

**Comment by philnash** [comment]
Hey @omnilord. Thanks very much for the detailed report. I have reproduced the issue. I imagine it's something to do with getting the correct files in the load path.

`envyable install` is fairly simple, all it does is add a directory called `config` and then two files `config/env.yml` and `config/env.yml.example`. It then adds `config/env.yml` to your `.gitignore` file if you have one.

I'll try to get this issue sorted soon. I'd love to know what the other problem is, please let me know as soon as you can 🙂

**Comment by philnash** [comment]
This should be fixed in envyable 1.1.1 which is now available.

**Comment by omnilord** [comment]
Actually, this fixed my other problem that involved values being cached indefinitely after installing Envyable.  Whatever values were in ```config/env.yml``` when Envyable was initialized the very first time were the only values to ever get imported into ENV by Envyable.  Changing values in ```config/env.yml``` never updated ENV regardless of actions (such as restarting rails server, re-opening terminals, etc.).  The only way to refresh the ENV values from ```config/env.yml``` was to uninstall Envyable, make the changes, then reinstall Envyable when the values were correct.

This is no longer a problem in version 1.1.1.  Thank you for the update.
